rocksdb_replicator/tests/rocksdb_assumption_test.cpp

```diff
@@ -222,11 +222,16 @@ TEST(RocksDBAssumptionTest, SequenceNumber) {
 
 
   // add two SST files to the empty DB, the sequence # should still be zero
```

Comment:
Can we add a test to verify what happens to sequence # after ingesting files into a non-empty DB? Our cluster management script/task may need to be modified accordingly.